frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -188,6 +195,34 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
             error: undefined,
             annotations,
           });
+
+          /**Presence/Absence mode */
+          if (!!this.state.task && this.state.task.annotationScope === SCOPE_WHOLE) {
+            let useTags = [];
+            let doYouSearchFirstAnnotationTypeTag = true;
+            annotations.map((annotation) => {
+              if (doYouSearchFirstAnnotationTypeTag && annotation.type === TYPE_TAG) {
+                annotation.active = true;
+                doYouSearchFirstAnnotationTypeTag = false;
+              }
+              !useTags.includes(annotation.annotation) && useTags.push(annotation.annotation);
+              return annotation;
+            })
+
+            useTags.forEach(element => {
+              const tagIndex = this.state.task.annotationTags.indexOf(element);
+              const checkbox_btn = document.querySelector(`#tags_key_checkbox_shortcuts_${tagIndex}`)
+              const btn_tag = document.getElementById(`tags_key_shortcuts_${tagIndex}`);
+
+              checkbox_btn.checked = true;
+              btn_tag.disabled = false;
```

Comment:
This is not the way to do such things in React. React creates a "virtual DOM" and updates regularly the "real" DOM (the one available through `document.querySelector()` or `document.getElementById()`). By using these functions, you override this mecanism and risk to see your modifications lost. The good way to handle those things is to update the state (here, set an annotation with a given tag). The modification of the state will trigger a re-render of the page, so you must modify the  `renderTag()` function where you render these buttons, and so have access through React to the `checked` and `disabled` attributes.

You have used the same method on these lines: 276 > 277, 290 > 292, 467 > 469, 485 > 486, 503 > 504, 535 > 539.